### PR TITLE
Add information about steps required before importing your migrated data.

### PR DESCRIPTION
**Pull request by kwacky1**
### Why:

If you land on this page from a search, there is no indication that prior steps are required.

### What's being changed:

A small preamble is added before the process that mentions and links to the previous steps

### Check off the following:
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

File: content/admin/user-management/migrating-data-to-your-enterprise.md [hunk 1]
```diff
@@ -17,10 +17,12 @@ versions:
 
 ### Applying the imported data on {% data variables.product.prodname_ghe_server %}
 
+Once you have [prepared your migration](/admin/user-management/preparing-to-migrate-data-to-your-enterprise) you can use the following steps to complete the migration.
+
 {% data reusables.enterprise_installation.ssh-into-target-instance %}
 
 2. Using the `ghe-migrator import` command, start the import process. You'll need:
-    * Your Migration GUID.
+    * Your [Migration GUID](/admin/user-management/preparing-to-migrate-data-to-your-enterprise).
```
Comment (felicitymay) [inline content/admin/user-management/migrating-data-to-your-enterprise.md:25]: To simplify localization, wherever possible we avoid inline links.

```suggestion
    * Your Migration GUID. For more information, see "[Preparing to migrate data to your enterprise](/admin/user-management/preparing-to-migrate-data-to-your-enterprise)."
```

**Review by felicitymay — CHANGES_REQUESTED** [review]
Thanks for raising this PR. It's a good call to add this extra context to the start of this procedure for people who arrive from a search ✨ 

There's one change needed to bring this into line with our localization guidelines. Otherwise this looks like a great change.

You can see your changes on staging here: https://docs-4007--kwacky1-migration-c.herokuapp.com/en/enterprise-server@3.0/admin/user-management/migrating-data-to-your-enterprise.

**Comment by sartgoo** [comment]
team-review-requested:[TEAMNAME4007[USERNAME]

**Review by felicitymay — APPROVED** [review]
Thanks. This looks ready to merge once the repository is unfrozen (should be today).

**Pull request merged**